### ISSUE TITLE
pytest - zeep - DeprecationWarning : defusedxml.lxml is no longer supported and will be removed in a future release

### ISSUE THREAD

**Issue by andreabisello** [mvantellingen/python-zeep#1014]
Python 3.7.4
Pytest 5.2.1
Zeep 3.2

code to reproduce

```
from zeep import Client

def test_number_to_words():
    client = Client('http://www.dataaccess.com/webservicesserver/numberconversion.wso?WSDL')
    assert client.service.NumberToWords(100) == "one hundred "

```

on execution result

```
============================== warnings summary ===============================
C:\Users\andreabisello\AppData\Local\Programs\Python\Python37-32\lib\site-packages\zeep\loader.py:3
  C:\Users\andreabisello\AppData\Local\Programs\Python\Python37-32\lib\site-packages\zeep\loader.py:3: DeprecationWarning: defusedxml.lxml is no longer supported and will be removed in a future release.
    from defusedxml.lxml import fromstring

-- Docs: https://docs.pytest.org/en/latest/warnings.html

```

the warning looks related to pytest, but about zeep.

due i want to use pytest and zeep in production environment, should i move to another test framework or another soap library?

thanks.

**Comment by ShaheedHaque** [comment]
I'm seeing this deprecation warning filling our logs too.

**Comment by arjitkgupta** [comment]
I am also getting the same error. 

**DeprecationWarning: defusedxml.lxml is no longer supported and will be removed in a future release.**

Is this issue resolved ?

I am using below. 
Python 3.6.1
pytest 5.4.2
zeep 3.4.0

**Comment by xmo-odoo** [comment]
Considering there are 2 issues which are still open, 2 PRs have been closed without merging, and 3.4.0 is the latest release to date... do you really have to ask?
